python/py-igraph/Portfile

```diff
@@ -77,20 +75,22 @@ if {${name} ne ${subport}} {
         build.env-append        IGRAPH_CMAKE_EXTRA_ARGS=[join $extra_cmake_args]
     }
 
-    # python-igraph optionally makes use of these, and some tests depend on them.
-    # If they are not installed, the corresponding tests will simply be skipped..
-    depends_test-append     port:py${python.version}-matplotlib \
-                            port:py${python.version}-networkx \
-                            port:py${python.version}-numpy \
-                            port:py${python.version}-pandas \
-                            port:py${python.version}-scipy
+    # Testing py312-igraph disabled until all test dependencies get py312 subports.
+    if {${subport} ne "py312-${python.rootname}"} {
```

Comment:
I think that this is the remaining concern:

[Previously, you wrote](https://github.com/macports/macports-ports/pull/20850#issuecomment-1768729733) “I do not have the time to keep checking when functionally irrelevant dependencies get updated.” I don’t think that’s what’s being requested. This port (`py-igraph`) is equipped to run all tests, including some tests that “optionally make[s] use of these” modules. For the purpose of testing, these modules _are_ relevant. All that’s being requested is that you do the same for the `py312-igraph` subport as is done for other Python versions, so that all tests can be run there too.

Other `py312-` subports are awaiting `py312-pytest` and its `py312-` dependencies, so I think that if the `py312-igraph` test dependency situation can’t be resolved on the speedier side, the `py312-pytest` should be broken out into a separate change.

Reply:
> All that’s being requested is that you do the same for the py312-igraph subport as is done for other Python versions, so that all tests can be run there too.

With all due respect, as the maintainer of this port, the decision about enabling or disabling should be mine, and I am not willing to delay the update of py-igraph until it can be tested _within MacPorts_ with Python 3.12. I am also a member of the igraph team, and I am confident that py-igraph works with Python 3.12. We test it regularly.

@reneotten, I made the changes you requested several days ago. This PR has been open for a week. Is there anything else you require before merging can be approved? I want to get this off my table and move forward.